sync/upstream.py

```diff
@@ -339,6 +339,7 @@ def create_pr(self):
             time.sleep(1)
 
         commit_summary = self.wpt_commits[0].commit.summary
+        commit_summary = commit_summary.rstrip("!#$%&(*+,-/:;<=>@[\\^_`{|~")
```

Comment:
I just used whatever is in `string.punctuation` minus quotes, closing parens, period, ?

Reply:
It's probably not unicode correct, but it's also probably good enough :)

But this only updates the GH summary, not the actual commit message, right?

Reply:
Oh, oops, I thought only the PR title was upsetting. Will take another look.